common/utils/src/main/resources/error/error-classes.json

```
@@ -3004,6 +3004,12 @@
     ],
     "sqlState" : "2200E"
   },
+  "NULL_QUERY_STRING_EXECUTE_IMMEDIATE" : {
+    "message" : [
+      "Execute immediate requires non-null value for sqlString variable, but provided <varName> is null."
```

Is it clear from user perspective what's `sqlString` is?

How about:
```suggestion
      "Execute immediate requires a non-null variable as its argument, but the provided variable <varName> is null."
```

---

or
```suggestion
      "Execute immediate requires a non-null variable as the query string, but the provided variable <varName> is null."
```